explainer-client/src/main/scala/ExplainEditorJS.scala

```diff
@@ -37,86 +37,135 @@ object Ajaxer extends autowire.Client[Js.Value, Reader, Writer]{
 object ExplainEditorJS {
 
   val endpoint = "wss://presence.code.dev-gutools.co.uk/socket"
-  val person = new Person("A","Nother","a.nother.@guardian.co.uk")
-  val presenceClient = PresenceGlobalScope.presenceClient(endpoint, person)
 
   object Model {
     import org.scalajs.jquery.{jQuery => $}
 
-    val explainer = Var(Explainer)
+    val explainer = Var(ExplainerItem)
 
-    def init(id: String): Future[Explainer] = {
+    def extractExplainer(id: String): Future[ExplainerItem] = {
       Ajaxer[ExplainerApi].load(id).call()
     }
 
-    def saveContent(id: String, explainer: ExplainerUpdate) = {
+    def updateFieldContent(id: String, explainer: ExplainerUpdate) = {
       Ajaxer[ExplainerApi].update(id, explainer.field, explainer.value).call()
     }
 
+    def createNewExplainer() = {
+      Ajaxer[ExplainerApi].create().call()
+    }
+
+    def publish(id: String): Future[ExplainerItem] = {
+      Ajaxer[ExplainerApi].publish(id).call()
+    }
+
   }
 
-  val headline: TypedTag[Input] = input(
-    id:="new-atom",
-    placeholder:="headline",
-    autofocus:=true
-  )
+  def presenceWrapper(explainerId: String ,area: String, field: Element,presenceClient: PresenceClient) = {

```

Comment:
Yes, but then you explain t me that "wrapper" is what is usually used in those cases :)